extensions/functions_string.yaml

```diff
@@ -1,6 +1,31 @@
 %YAML 1.2
 ---
 scalar_functions:
+  -
+    name: ascii
+    description: Returns the numeric value of the first character of `input`.
```

Comment:
Could you also give more details about the UTF8 option?  The first 128 characters match up with ascii, but what happens after that when the input is UTF8?